middleware.py

```diff
@@ -0,0 +1,25 @@
+import jwt
+
+from django.conf import settings as django_settings
+
+
+class UserSessionSharingMiddleware(object):
+    """
+    Middleware to set jwt login token on sign in
+    Used for session sharing with NodeBB community
+    """
+
+    def process_response(self, request, response):
+        try:
+            if request.user.is_authenticated():
+                encoded_jwt = jwt.encode({'id': request.user.id,
+                                          'username': request.user.username,
+                                          'email': request.user.email},
+                                         django_settings.OPENEDX_NODEBB_DISCUSSION['SECRET'],
```

Comment:
`SECRET` is not that descriptive of a name. Use something more descriptive like `DISCUSSION_JWT_SECRET`

Reply:
Done